connector.go

```diff
@@ -17,7 +17,8 @@ type Connector interface {
 	Call(functionName string, args interface{}) (resp *Response, err error)
 	Call17(functionName string, args interface{}) (resp *Response, err error)
 	Eval(expr string, args interface{}) (resp *Response, err error)
-	Execute(expr string, args interface{}) (resp *Response, err error)
+	Execute(expr interface{}, args interface{}) (resp *Response, err error)
+	Prepare(expr string) (resp *Response, err error)
```

Comment:
Do we need `Unprepare` function?
https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_sql/prepared_table/#box-sql-box-unprepare-prepared-statement

Reply:
Sure, thank you for highlighting it!

Reply:
Unprepare added.

Reply:
Added.

Reply:
I still don't see it in `connector.go`